app/utils/format_date.py

```diff
@@ -0,0 +1,24 @@
+from datetime import datetime
+
+CRYSTAL_REPORT_EXTRACT_FORMAT = "%dM%mM%Y"
+
+def _to_format(date_str: str, desired_format: str) -> str | None:
```

Comment:
I don't think we need parts of this module which are pilot data specific.

Reply:
It was good to talk through this, sticking to the expected NHS Notify date formats is less internal logic and is conveniently documented. It will make validation of inbound requests easier for us as we may be able to use the same schema/OAS file to validate post data.